diff --git a/dist/module/lib/evaluation/handlers/uriEvaluationHandler.js b/dist/module/lib/evaluation/handlers/uriEvaluationHandler.js
index cbb80b6227d3db4cd3829a07bd376a9ef9d0b0e5..7f1c542296422a6ce0be1ef39e0ae7a78f84ce52 100644
--- a/dist/module/lib/evaluation/handlers/uriEvaluationHandler.js
+++ b/dist/module/lib/evaluation/handlers/uriEvaluationHandler.js
@@ -5,8 +5,10 @@ import { Status } from '../../ConstraintUtils';
 import { PEVersion } from '../../types';
 import PexMessages from '../../types/Messages';
 import { AbstractEvaluationHandler } from './abstractEvaluationHandler';
+import matchAll from 'string.prototype.matchall';
+
 export class UriEvaluationHandler extends AbstractEvaluationHandler {
-    static matchAll = require('string.prototype.matchall');
+    static matchAll = matchAll;
     constructor(client) {
         super(client);
     }
diff --git a/dist/module/lib/utils/jsonPathUtils.js b/dist/module/lib/utils/jsonPathUtils.js
index 60ce3b2a5423357c3808015221383aeee3a6f6b4..46593d83f366ecf04554f3c5430fe7d5b526686b 100644
--- a/dist/module/lib/utils/jsonPathUtils.js
+++ b/dist/module/lib/utils/jsonPathUtils.js
@@ -1,6 +1,8 @@
 import { JSONPath as jp } from '@astronautlabs/jsonpath';
+import matchAll from 'string.prototype.matchall';
+
 export class JsonPathUtils {
-    static matchAll = require('string.prototype.matchall');
+    static matchAll = matchAll;
     static REGEX_PATH = /@\w+/g;
     /**
      * @param obj: any object can be found in verifiablePresentation.verifiableCredential[i]
